content/en/docs/installing-on-linux/introduction/intro.md

```diff
@@ -81,13 +60,9 @@ KubeSphere has decoupled some core feature components since v2.1.0. These compon
 
 ## Storage Configuration Instruction
 
-The following links explain how to configure different types of persistent storage services. Please refer to [Storage Configuration Instruction](../storage-configuration) for detailed instructions regarding how to configure the storage class in KubeSphere.
+KubeSphere allows you to configure persistent storage services before installation and after installation. Meanwhile KubeSphere supports a varity of storage services including open source solutions such as Ceph, GlusterFS, etc. and commercial storage products as well. Please refer to [Persistent Storage Configurations](../storage-configuration) for detailed instructions regarding how to configure the storage class before installing KubeSphere.
 
-- [NFS](https://kubernetes.io/docs/concepts/storage/volumes/#nfs)
-- [GlusterFS](https://www.gluster.org/)
-- [Ceph RBD](https://ceph.com/)
-- [QingCloud Block Storage](https://docs.qingcloud.com/product/storage/volume/)
-- [QingStor NeonSAN](https://docs.qingcloud.com/product/storage/volume/super_high_performance_shared_volume/)
+As said, You can set different storage classes after installing KubeSphere which can be used for your workloads. For more information, see [Add New Storage Classes](../../../cluster-administration/persistent-volume-and-storage-class/).
```

Comment:
->
For more information about how to set different storage classes for your workloads after you install KubeSphere, see [Add New Storage Classes](../../../cluster-administration/persistent-volume-and-storage-class/).

Reply:
done